src/monitor/monitor.c

```diff
@@ -405,17 +405,34 @@ static int add_svc_conn_spy(struct mt_svc *svc)
 
 static void svc_child_info(struct mt_svc *svc, int wait_status)
 {
+    int exit_code = 0;
+    int pid = svc->pid;
+    const char *name = (svc->name ? svc->name : "");
+    const char *identity = (svc->identity ? svc->identity : "");
+
     if (WIFEXITED(wait_status)) {
-        DEBUG(SSSDBG_OP_FAILURE,
-              "Child [%d] exited with code [%d]\n",
-               svc->pid, WEXITSTATUS(wait_status));
+        exit_code = WEXITSTATUS(wait_status);
+        if (exit_code == SSS_WATCHDOG_EXIT_CODE) {
+            DEBUG(SSSDBG_CRIT_FAILURE,
+                  "Child [%d] ('%s':'%s') was terminated by own WATCHDOG\n",
```

Comment:
Why ('%s':'%s') instead of just (%s:%s) ?

Reply:
Name/identity may contain weird symbols, including `:`. Having it wrapped in `' '` improves readability from my point of view.

Reply:
OK, agree. Just one question - why single quoetes '_' instead of double quotes "_" which are more common for C based software?

Reply:
Because you don't have to bother with escaping them :-)

Reply:
I don't have strong preference here.

Reply:
I agree to what you proposed here